dodola/tests/test_core.py

```diff
@@ -44,6 +44,36 @@ def _train_simple_qdm(
     return qdm
 
 
+def test_train_quantiledeltamapping_quantiles_excludeendpoints():
+    """Test that "endpoints" are not included in trained QDM models"""
+    n_quantiles = 100
+    n_days = 365
+    n_years = 20
+
+    # Make up some data to training on...
+    variable_kind = "+"
```

Comment:
I think we should probably pass this in as a parameter and test for both additive and multiplicative QDM models rather than hardcoding `kind` here...

Reply:
Changed in 9fad7d5. Marking resolved.